gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/tasks/PlantUmlTask.groovy

```diff
@@ -81,6 +81,18 @@ class PlantUmlTask extends AbstractTask implements Specification {
 		targetFile.write(template.toString(), 'UTF-8')
 	}
 
+	private URL getTemplate() {
+
+		String resource
+		if (project.restApi.diagramShowFields) {
```

Comment:
What is the reason for two templates. Can that behaviour _Show fields_ not be with the single template?